<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPanel</class>
 <widget class="QWidget" name="AdminPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton_SignOut">
     <property name="text">
      <string>SignOut</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>add user</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" rowspan="2" colspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="3" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>username:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEdit_addUser"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>password:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEdit_addPass">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QPushButton" name="pushButton_addUser">
             <property name="text">
              <string>Add User</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QComboBox" name="comboBox_addUser">
             <item>
              <property name="text">
               <string>user</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>admin</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>power :</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_10">
             <property name="styleSheet">
              <string notr="true">color:red;
font-size:15px;</string>
             </property>
             <property name="text">
              <string>برای حذف کاربر تیک کنارش را بزنید</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_11">
             <property name="styleSheet">
              <string notr="true">color:red;
font-size:15px;</string>
             </property>
             <property name="text">
              <string>دقت کنید بعد از ویرایش کاربر حتما دکمه submit edit را بزنید</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_12">
             <property name="styleSheet">
              <string notr="true">color:red;
font-size:15px;</string>
             </property>
             <property name="text">
              <string>از سمت چپ در هر ردیف ابتدا نام کاربری و بعد از کاما رمز عبور است.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QListWidget" name="listWidget_users"/>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="pushButton_submitUserEdit">
               <property name="text">
                <string>Submit Edit</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_refreshUserList">
               <property name="text">
                <string>Refres User list</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Book Management</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1" rowspan="2">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QFormLayout" name="formLayout_2">
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">color:rgb(255, 0, 0);
font-weight:800;
font-size:15px;</string>
             </property>
             <property name="text">
              <string>نکته : 1-برای حذف کتاب تیک کنار کتاب را بزنید</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_5">
             <property name="styleSheet">
              <string notr="true">color:rgb(255, 0, 0);
font-weight:800;
font-size:14px;</string>
             </property>
             <property name="text">
              <string>2- برای ویرایش کتاب دو بار بر روی کتاب کلیک کنید پس از ویرایش دکمه ی submit edit  را بزنید. </string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>book name:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="lineEdit_bookName"/>
           </item>
           <item row="8" column="1">
            <widget class="QPushButton" name="pushButton_addBook">
             <property name="text">
              <string>Add Book</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QLineEdit" name="lineEdit_authorName"/>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Author name:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_8">
             <property name="styleSheet">
              <string notr="true">color:rgb(255, 0, 0);
font-weight:800;
font-size:15px;</string>
             </property>
             <property name="text">
              <string>برای اطمینان از ویرایش . لیست کتاب هارا رفرش کنید</string>
             </property>
            </widget>
           </item>
           <item row="7" column="1">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Search</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_13">
             <property name="styleSheet">
              <string notr="true">color:rgb(255, 0, 0);
font-weight:800;
font-size:14px;</string>
             </property>
             <property name="text">
              <string>برای جستجوی کتاب نام ان را وارد کرده و روی Search کلیک کنید</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="label_14">
             <property name="styleSheet">
              <string notr="true">color:rgb(255, 0, 0);
font-weight:800;
font-size:14px;</string>
             </property>
             <property name="text">
              <string>از سمت چپ در هر ردیف ابتدا نام کتاب و بعد از کاما نام نویسنده است.</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Bookname  , Authorname</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_books"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButton_submitEdit">
             <property name="text">
              <string>Submit Edit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_refreshBookList">
             <property name="text">
              <string>Refresh Book list</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="3">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
